templates/frontend/components/searchForm_simple.tpl

```diff
@@ -10,7 +10,13 @@
  * @uses $searchQuery string Previously input search query
  *}
 {if !$currentJournal || $currentJournal->getData('publishingMode') != \APP\journal\Journal::PUBLISHING_MODE_NONE}
-<form class="navbar-form navbar-left" role="search" method="post" action="{url page="search" op="search"}">
+{capture name="searchFormUrl"}{url page="search" op="search" escape=false}{/capture}
+{$smarty.capture.searchFormUrl|parse_url:$smarty.const.PHP_URL_QUERY|parse_str:$formUrlParameters}
+<form class="navbar-form navbar-left" role="search" method="get" action="{$smarty.capture.searchFormUrl|strtok:"?"|escape}">
+  {csrf}  
```

Comment:
The search form shouldn't include a CSRF token (it's intentionally not checked in the public search form).